.gitmodules

```diff
@@ -80,3 +80,6 @@
 [submodule "platform/p4/sonic-platform-modules-arista"]
 	path = platform/p4/sonic-platform-modules-arista
 	url = https://github.com/aristanetworks/sonic
+[submodule "platform/barefoot/sonic-platform-modules-ingrasys"]
```

Comment:
no longer accepting submodules for platform modules, all platform modules source code should be check-in into the sonic-buildimage. we will also enforce this rule for existing platform modules. this will make kernel upgrade easier in the future.